### ISSUE TITLE
Test Failed on mvn package

### ISSUE THREAD

**Issue by satoshikumano** [swagger-api/swagger-codegen#7979]
##### Description

After cloning repository, `mvn clean package` failed because of the test failure.

```
Tests run: 582, Failures: 2, Errors: 0, Skipped: 3, Time elapsed: 20.075 sec <<< FAILURE! - in TestSuite
checkOptionsHelp(io.swagger.codegen.spring.SpringOptionsTest)  Time elapsed: 0.022 sec  <<< FAILURE!
java.lang.NullPointerException

checkOptionsProcessing(io.swagger.codegen.spring.SpringOptionsTest)  Time elapsed: 0.002 sec  <<< FAILURE!
java.lang.NullPointerException


Results :

Failed tests: 
  SpringOptionsTest>AbstractOptionsTest.checkOptionsHelp:39 » NullPointer
  SpringOptionsTest>AbstractOptionsTest.checkOptionsProcessing:28 » NullPointer
```

##### Swagger-codegen version

40d5d0990501964d0547aaeb591be4f8276379af

##### Swagger declaration file content or url

None.

##### Command line used for generation

`mvn clean package`

maven: 
```
satoshi$ mvn -version
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-11T01:41:47+09:00)
Maven home: /usr/local/Cellar/maven/3.3.9/libexec
Java version: 1.8.0_162, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_162.jdk/Contents/Home/jre
Default locale: en_JP, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.2", arch: "x86_64", family: "mac"
```

##### Steps to reproduce

```
git clone git@github.com:swagger-api/swagger-codegen.git
cd swagger-codegen.git
mvn clean package
```

##### Related issues/PRs

Could not find similar failure.

##### Suggest a fix/enhancement

I added `-DskipTests` to avoid the failure. (Workaround)

```
mvn package -DskipTests
```

**Comment by wing328** [comment]
@satoshikumano thanks for reporting the issue. Please use "mvn clean package -DskipTests" as you've mentioned for the time being.

**Comment by wing328** [comment]
@satoshikumano the issue should be fixed via #7983 

Please pull the latest master to give it another try. Thanks again for reporting the issue.
